### DISCUSSION TITLE
adding one more dimension to gluonts deepVAR gives a infer_shape error

### DISCUSSION THREAD

**Discussion by ShirleyMgit** [awslabs/gluonts#2699] [Q&A]
I have trained gluonts deepAR and deepVAR and I would like to get the predicted distribution parameters without sampling,  is it possibe?

**Comment by mbohlkeschneider** [comment]
Hi @ShirleyMgit,

Thank you for your interest in using GluonTS. I'm not sure I fully understand the question. To clarify, the distribution parameters are neural network outputs, so you can only get them but unrolling the network over some input data. You can check [this script](https://github.com/mbohlkeschneider/gluon-ts/blob/mv_release/test/model/multivariate/test_covariance.py) on how it works in general, but you will need to invoke `predict` (and basically sample) still to do the forward pass to compute the distribution parameters. 

Hope that helps!
